### PR TITLE
Admin changes

### PR DESCRIPTION
**Pull request by ldelafue**
I added the information about the new online system

### CODE REVIEW

**Comment by ldelafue** [comment]
I did the change in delegates and travel assistant

**Comment by ldelafue** [comment]
and now

File: docs/admin/online_travel_system.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Signing up the travel system account
+
+You must be registered in the system for paperwork related with travel authorizations and travel reimbursements. Booking flighs and lodging can be done in the same system too but in general it is more expensive. Please check in the designed hotels by the conference and in alternative services.
```
Comment (welytle) [inline docs/admin/online_travel_system.md:3]:
```suggestion
You must be registered in the system for paperwork related with travel authorizations and travel reimbursements. Booking flights and lodging can be done in the same system too but in general it is more expensive. Please check in the designated hotels by the conference and in alternative services.
```

File: docs/admin/online_travel_system.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+### To activate your account:
+1. Visit [https://travel.financialservices.arizona.edu/] (https://travel.financialservices.arizona.edu/)
+2. Once logged in, click on the profile botton (blue circle in the upper right corner)
```
Comment (welytle) [inline docs/admin/online_travel_system.md:7]:
```suggestion
2. Once logged in, click on the profile button (blue circle in the upper right corner)
```

File: docs/admin/travel_autorization.md [hunk 3]
```diff
@@ -1,13 +1,22 @@
...
-2. The form must be downloaded always from [https://financialservices.arizona.edu/form/travel-authorization-form](https://financialservices.arizona.edu/form/travel-authorization-form) because it has a unique correlative number.
-3. You must save the form into your device before filling it out. If you complete the form in your web browser data will be lost.
+2. The form must be filled in the online system. [https://travel.financialservices.arizona.edu/] (https://travel.financialservices.arizona.edu/). Remember you must be registerd in the system.
+
+
+### Create Travel Request:
+1. To create a request from the homepage click New (top + next to University of Arizona) and choose Start a Request and fill in all the required fields.
+2. Add the Bussines Travel dates with the maximum extension you think the conference would last. Include personal time on that.
+3. Each time the conference has Lodging options you should add this to the form. You can use another later explaining the reasons.
```
Comment (welytle) [inline docs/admin/travel_autorization.md:10]:
```suggestion
3. Each time the conference has lodging options you should add this to the form. You can use another later explaining the reasons.
```

File: docs/admin/travel_autorization.md [hunk 4]
```diff
@@ -1,13 +1,22 @@
...
+4. You have to explain the reasons of the conference, academic development, paper presentation, University representation, etc. Add a small detail about how the trip would be payed.
+5. The organization must (0469) Hydrology and Atmospheric Sci.
```
Comment (welytle) [inline docs/admin/travel_autorization.md:12]:
```suggestion
5. The organization must be (0469) Hydrology and Atmospheric Sci.
```

**Review by welytle — COMMENTED** [review]
I left some small suggestions for typo fixes. Would mind updating these?

**Review by welytle — APPROVED** [review]
Looks good. Thanks for adding this in!

**Pull request merged**